Automattic/jetpack
# Share the block style selector code

Fixes #14466 

#### Changes proposed in this Pull Request:

* OpenTable and Calendly both have a block style selector which does essentially the same thing. Let's extract that to a new shared component.

#### Testing instructions:






* Add a Calendly block and check that you can still change the styles in the sidebar
* Add an OpenTable block and check that you can still change the styles in the sidebar
<img width="308" alt="Screenshot 2020-01-17 at 13 48 58" src="https://user-images.githubusercontent.com/275961/72617078-23334f00-3930-11ea-853b-fe7569bf695b.png">
<img width="319" alt="Screenshot 2020-01-17 at 13 48 51" src="https://user-images.githubusercontent.com/275961/72617081-23cbe580-3930-11ea-9ca4-cfe9fcfbf07d.png">


#### Proposed changelog entry for your changes:

* no changelog
